docs/deployment/security/security-kerberos.md

```diff
@@ -87,15 +87,18 @@ Steps to run a secure Flink cluster in standalone/cluster mode:
 2. Ensure that the keytab file exists at the path indicated by `security.kerberos.login.keytab` on all cluster nodes.
 3. Deploy Flink cluster as normal.
 
-### YARN/Mesos Mode
+### YARN, Mesos and native Kubernetes Mode
```

Comment:
nit: I would suggest prioritizing native k8s over yarn/mesos.

Reply:
And if you search the keyword `YARN` in this file, you'll find a few other places where yarn and mesos are mentioned but k8s is missing.